app/client/src/features/dashboard/DashboardEventListItem.tsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { useRouter } from 'next/router';
+import { Button, ListItem, ListItemSecondaryAction } from '@mui/material';
+import { DashboardEvent } from '@local/features/dashboard/DashboardEvent';
+
+interface DashboardEventListItemProps {
+    event: {
+        id: string;
+        title: string | null;
+        description: string | null;
+        startDateTime: Date | null;
+        endDateTime: Date | null;
+        organization: {
+            name: string;
+        } | null;
+    };
+    ongoing: boolean;
+    divider: boolean;
+}
+
+export function DashboardEventListItem({ event, ongoing, divider }: DashboardEventListItemProps) {
+    const router = useRouter();
+    const handleNav = (path: string) => () => router.push(path);
+
+    return (
+        <ListItem key={event.id} divider={divider} button onClick={handleNav(`/events/${event.id}/settings`)}>
+            {event.title && event.description && event.startDateTime && event?.organization?.name && (
+                <DashboardEvent
+                    key={event.id}
+                    id={event.id}
+                    title={event.title}
+                    description={event.description}
+                    startDateTime={event.startDateTime}
+                    organization={event?.organization?.name}
+                />
+            )}
+
+            {ongoing && (
+                <ListItemSecondaryAction>
+                    <Button
+                        aria-label='view live feed of current event'
+                        variant='contained'
+                        color='primary'
+                        onClick={handleNav(`/events/${event.id}/live`)}
+                    >
+                        Live Feed
+                    </Button>
+                </ListItemSecondaryAction>
```

Comment:
I think potentially changing the API of this would help a little with extendability in a general sense.

Maybe instead allow for a children to be passed in so that way the caller can define what type of button to use and the DashboardEventListItem doesn't need to know if it's ongoing or not. So then the caller does
```tsx
<DashboardEventListItem>
  <ListItemSecondaryAction></ListItemSecondaryAction>
</DashboardEventListItem>
```

and then in this file this code just becomes `{children}` which you can destructure from the props.

Then, the caller, which always will know the type of events, will be able to customize the list item as necessary.